code/lib/cli/src/detect.ts

```diff
@@ -106,23 +106,29 @@ export function detectFrameworkPreset(
  *
  * @returns CoreBuilder
  */
-export function detectBuilder(packageManager: JsPackageManager) {
+export function detectBuilder(packageManager: JsPackageManager, projectType: ProjectType) {
   const viteConfig = findUp.sync(viteConfigFiles);
-
   if (viteConfig) {
     paddedLog('Detected vite project, setting builder to @storybook/builder-vite');
     return CoreBuilder.Vite;
   }
 
-  const nextJSVersion = detectNextJS(packageManager);
-  if (nextJSVersion) {
-    if (nextJSVersion >= 11) {
-      return CoreBuilder.Webpack5;
```

Comment:
I guess this was probably here to choose between webpack 4 and 5 originally.  I think it makes sense to remove it as you have.  :+1: